src/nix/version.rs

```diff
@@ -0,0 +1,94 @@
+//! Rust module for `nix --version`
+use leptos::*;
+use regex::Regex;
```

Comment:
Since this import is only used a function that is available server-side only:

```suggestion
#[cfg(feature = "ssr")]
use regex::Regex;
```

Reply:
> I have a question:
> 
> Why am I getting this warning even though I am using `Regex` in `parse_nix_version` function?

Because of this ^